insights/specs/default.py

```diff
@@ -1035,6 +1035,7 @@ def xfs_mounts(broker):
     xfs_info = foreach_execute(xfs_mounts, "/usr/sbin/xfs_info %s")
     xinetd_conf = glob_file(["/etc/xinetd.conf", "/etc/xinetd.d/*"])
     yum_conf = simple_file("/etc/yum.conf")
+    yum_list_available = simple_command("yum --noplugins list available")
```

Comment:
May I know why you remove the `-C` here? `-C` is added to avoid updating the cache, since the cache-update needs some network traffics, this issue has been complained by some CUs.

Reply:
Since this spec is a `simple_command`, it will be collected into the insights-archive as a `simple_file`. Another entry for this spec needs to be added into the `insights/spec/insights_archive.py`

See: https://github.com/RedHatInsights/insights-core/blob/495c0985a005edb807c41231c9748898a2bc984f/insights/specs/insights_archive.py#L296

Reply:
> May I know why you remove the `-C` here? `-C` is added to avoid updating the cache, since the cache-update needs some network traffics, this issue has been complained by some CUs.

The motivation behind this parser is to enable a rule that will check if there are available kernel live patch modules (kpatches) that the user has not already installed. The mechanism for delivery of these modules is via the "kpatch-patch" rpm. kpatch is generally used to deliver critical security patches to systems that must maintain uptime, so by not updating the cache, we may end informing a user that there are no new critical available security patches available when in fact there are, leaving the user vulnerable.

So to make sure we don't give the user any false negative results, I update the cache.